src/utils/mapper/invitation.js

```diff
@@ -0,0 +1,9 @@
+export const fromInvitationDtoToInvitation = (invitation, event) => ({
+  id: invitation.id,
+  event: { ...event, id: event._id },
```

Comment:
i thought event objects would already have an id?

Reply:
This function won't be necessary if invitation resolver is implemented as commented above